### PR TITLE
[SBS-34][SBS-43] component: add custom header, backbutton components

### PR DESCRIPTION
**Pull request by phdhuy**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

# Screenshot
Screenshot from 2024-05-18 22-16-08

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/components/BackButton.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <TouchableOpacity style={$iconContainer} onPress={() => navigation.goBack()}>
+      <Ionicons name="arrow-back" size={24} color={tintColor} />
+    </TouchableOpacity>
+  );
+};
+
+const $iconContainer: TextStyle = {
+  padding: 0,
+};
```
Comment (LeTatTuan) [inline app/components/BackButton.tsx]: TouchableOpacity  có padding mặc định hay răng set về 0 rứa mi.

**Comment by LeTatTuan** [comment]
Răng hình description có 2 headers rứa, thấy 3 icon chưa được thẳng hàng lắm.

File: app/components/BackButton.tsx (hunk 1 above)
Comment (phdhuy) [reply app/components/BackButton.tsx]: Đã xóa rồi nhé @LeTatTuan

**Comment by duchuyyyy** [comment]
> Răng hình description có 2 headers rứa, thấy 3 icon chưa được thẳng hàng lắm.

**Comment by phdhuy** [comment]
image
Center m

**Review by phamhongphuc1403 — APPROVED** [review]

**Review by LeTatTuan — APPROVED** [review]

**Pull request merged**